<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8CNA17E" failures="1" tests="1" name="StandaloneTest_2" time="9.286" errors="0" timestamp="02 Jan 2023 15:57:33 GMT">
  <testcase classname="testPackage.Tests.StandaloneTest_2" name="testMethod" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testPackage.Tests.StandaloneTest_2.testMethod(StandaloneTest_2.java:23)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
</testsuite> <!-- StandaloneTest_2 -->
